rust-mode.el

```diff
@@ -16,7 +16,8 @@
 
 (eval-when-compile (require 'rx)
                    (require 'compile)
-                   (require 'url-vars))
+                   (require 'url-vars)
+                   (require 'ansi-color))
```

Comment:
`ansi-color-apply-on-region` is a function, not a macro, so shouldn't this *not* be in `eval-when-compile`?